src/components/send-button/index.js

```diff
@@ -104,22 +105,28 @@ export default compose( [
 			publishStatus = 'publish';
 		}
 
+		const [ adLabel, setAdLabel ] = useState();
 		const [ adsWarning, setAdsWarning ] = useState();
+		const [ activeAdManageUrl, setActiveAdManageUrl ] = useState();
+
 		useEffect(() => {
 			apiFetch( {
-				path: `/wp/v2/${ NEWSLETTER_AD_CPT_SLUG }`,
+				path: `/wp/v2/${ NEWSLETTER_AD_CPT_SLUG }/count/?date=${ postDate }`,
 			} ).then( response => {
-				const activeAds = response.filter( isAdActive );
-				if ( activeAds.length ) {
+				const countOfActiveAds = response.count;
+				setActiveAdManageUrl( response.manageUrl );
+				setAdLabel( response.label );
+
+				if ( countOfActiveAds > 0 ) {
 					setAdsWarning(
-						sprintf(
+						sprintf( /* eslint-disable-line */
```

Comment:
Oh woops good catch. I had that there while developing. Removing.

Reply:
Ah, I couldn't get accommodate [this rule](https://github.com/WordPress/gutenberg/blob/trunk/packages/eslint-plugin/docs/rules/valid-sprintf.md) in a sane way, mainly because I'm not familiar with _n or _nx. I think I fixed that but then prettier doesn't like it : ). So I didn't remove it this round.

Reply:
I see now - there's actually an issue here and the rule should not be disabled. `_n` expects static strings as input, as this will be the content for translation - there can't be any interpolated variables there. So it should be:

```js 
sprintf(
	_n(
		'There is %d active %s.',
		'There are %d active %ss.',
		countOfActiveAds,
		'newspack-newsletters'
	),
	countOfActiveAds,
	response.label
)
```

Reply:
Fixed 👍